<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetThresholdDlg</class>
 <widget class="QDialog" name="SetThresholdDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>247</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox::up-button
{
	width: 35px;
	height: 25px;
}

QSpinBox::down-button
{
	width: 35px;
	height: 25px;
}</string>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="MyLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>最小值：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>247</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="minBox">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>80</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox::up-button
{
	width: 35px;
	height: 25px;
}

QSpinBox::down-button
{
	width: 35px;
	height: 25px;
}</string>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QLabel" name="titleLab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>阈值设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>207</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>统一设置</string>
   </property>
  </widget>
  <widget class="MyLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>90</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>最大值：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">setthreshold/mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
